New requre decorators for packit (#964)
packit/packit: tests_recording/README.md

contained different behaviour, so that we had to generate these response
files with various versions of these tools.

> TO find which tools needs some network access use commands like:

Reviewer: ```suggestion
The `unshare` command below will show tests which require network connectivity, because those will fail:
```

### Command to run

- Ensure to remove files and directories for tests what you would like to regenerate.
- you have to have configs
  - for packit: `~/.config/packit.yaml`
  - for copr: `~/.config/copr`

```
pytest-3 -v tests/testsuite_recording
```

#### Matrix

Install `requre` from git master branch

- now there are no issues with versions of various libraries

> - In case it happen, just regenerate data with proper versions and add there keys for versions

Reviewer: first sentence says there are no issues and the second tells what to do in case of issues - that doesn't make any sense :D 

also, what does "proper versions" and "add keys" mean exactly?

Author: fixed

### Command to run

- Ensure to remove files and directories for tests what you would like to regenerate.

> - you have to have configs

Reviewer 2: ```suggestion
- You have to have configs:
```

  - for packit: `~/.config/packit.yaml`
  - for copr: `~/.config/copr`

```

> pytest-3 -v tests/testsuite_recording

Reviewer 2: Isn't the right path this?
```suggestion
pytest-3 -v tests_recording
```

Author: right :-)

```

### Postprocessing

> - Remove secrets from stored files - Some parts are not covered by generic requre pre-commit hook for removing secrests, e.g. token and login for copr. Remove them manually or via requre-patch tools

Reviewer 2: ```suggestion
- Remove secrets from stored files - Some parts are not covered by generic requre pre-commit hook for removing secrests, e.g. token and login for copr. Remove them manually or via requre-patch tools:
```

  ```
  requre-patch purge --replaces "copr.v3.helpers:login:str:somelogin" --replaces "copr.v3.helpers:token:str:sometoken" tests_recording/test_data/*/*yaml
  ```

> - create symlinks for files in tests_data, `file` handler stores data to tar archives and more tests uses same archives, so you can symlink them via requre tool:

Reviewer 2: ```suggestion
- Create symlinks for same test-files in `tests_data` that are saved with the response. Requre uses tar archives for saving file content and you can easily symlink them via requre tool:
```

  ```
  requre-patch create-symlinks `pwd`/tests_recording/test_data/
  ```

#### Matrix

Install `requre` from git master branch

> - In current version of packit there are no issues with various versions libraries. In case it happens, regenerate data with proper versions of libraries and add there keys for these versions.

Reviewer 2: If I do this for the first time, I would not know what you mean by this. Maybe meantion what `key` are you talking about here.
